src/OpenTelemetry/Trace/CompositeActivityProcessor.cs

```diff
@@ -129,28 +129,30 @@ protected override bool OnForceFlush(int timeoutMilliseconds)
         }
 
         /// <inheritdoc/>
-        protected override void OnShutdown(int timeoutMilliseconds)
+        protected override bool OnShutdown(int timeoutMilliseconds)
         {
             var cur = this.head;
-
+            var result = true;
             var sw = Stopwatch.StartNew();
 
             while (cur != null)
             {
                 if (timeoutMilliseconds == Timeout.Infinite)
                 {
-                    cur.Value.Shutdown(Timeout.Infinite);
+                    result = cur.Value.Shutdown(Timeout.Infinite) && result;
```

Comment:
Is `&& result` necessary it defaults to `true` and `Showdown` should only be called once?

Reply:
This function has a for-loop which would go through a list of processors and call Shutdown from each of them.